scripts/linters/css_linter.py

```diff
@@ -58,6 +59,34 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(css_lint_output):
```

Comment:
If you're returning a string, call this get_trimmed_output() or get_trimmed_error_output(). Remember I mentioned that when you name something plural, it suggests a list.

Ditto elsewhere.

Reply:
Done.